packages/kit-headless/scripts/prepare-chrome-113.ts

```diff
@@ -0,0 +1,76 @@
+import os from 'os';
+import decompress from 'decompress';
+import fs from 'fs';
+
+let progressValue = 0;
+
+function printProgress(progress: number) {
+  if (progress % 5 === 0 && progressValue !== progress) {
+    console.log(`Download ${progress}% complete ⏬`);
+    progressValue = progress;
+  }
+}
+
+async function prepareChrome113() {
+  const path = `browsers/chrome/113/chrome-${os.platform()}`;
+
+  if (fs.existsSync(path)) {
+    console.log('Chrome 113 already exists. Skipping unzip ⏩');
+  } else {
+    console.log('Creating directory to save the browser 🌐');
+    fs.mkdirSync(path, { recursive: true });
+
+    switch (os.platform()) {
+      case 'linux':
+        console.log('Installing Chrome 113 for Linux 🐧');
+        break;
+
+      case 'darwin':
+        console.log('Installing Chrome 113 for macOS 🍎');
+        break;
+
+      case 'win32':
+        console.log('Installing Chrome 113 for Windows 🪟');
+        break;
+    }
+
+    const downloadResponse = await fetch(
+      `https://github.com/cwoolum/qwik-ui-polyfill-browsers/raw/main/chrome/113/chrome-${os.platform()}.zip`,
```

Comment:
I think this would be needed to change to `qwikifiers/`  right?

Reply:
This should be resolved now